package.json

```diff
@@ -41,7 +41,7 @@
   },
   "peerDependencies": {
     "intl": "^1.2.5",
-    "react-intl": "^2.8.0"
+    "react-intl": "^5.8.6"
```

Comment:
Should we bump the node min version to v10 with this MVB?

Reply:
Do we need to? In general I tend to only bump those up when we know theres a problem with the older version.. granted we don't exactly test against the older versions.